CHANGES.md

```diff
@@ -1,3 +1,9 @@
+## v0.7.1 (2022-03-14)
+
+* Fix syntax error in stubs ld file (#509)
```

Comment:
```suggestion
* Use `note.not_openbsd` instead of `note.not-openbsd` as section name in `solo5-stubs.ld` to be compatible with GNU ld 2.37 on e.g. Arch Linux, Fedora 35, Alpine 3.15 (@TheLortex, #509)
```